lib/tasks/tariff.rake

```diff
@@ -110,228 +78,20 @@ namespace :tariff do
     end
   end
 
-  namespace :install do
-    namespace :taric do
-      desc 'Add Sections and associate to Chapters'
-      task sections: :environment do
-        load(Rails.root.join('db/import_sections.rb'))
-      end
-
-      desc 'Dump Section notes'
-      task dump_section_notes: :environment do
-        SectionNote.all.each do |section_note|
-          section_file = "db/notes/sections/#{section_note.section_id}.yaml"
-          File.open(section_file, 'w') do |out|
-            section_doc = {
-              section: section_note.section_id,
-              content: section_note.content,
-            }
-            YAML.dump(section_doc, out)
-          end
-        end
-      end
-
-      desc 'Load Section notes into database'
-      task section_notes: :environment do
-        Dir[Rails.root.join('db/notes/sections/*')].each do |file|
-          note = YAML.safe_load(File.read(file), [Symbol])
-          section_note = SectionNote.find(section_id: note[:section]) || SectionNote.new(section_id: note[:section])
-          section_note.content = note[:content]
-          section_note.save
-        rescue StandardError => e
-          puts "Error loading: #{file}, #{e}"
-        end
-      end
-
-      desc 'Dump Chapter notes'
-      task dump_chapter_notes: :environment do
-        ChapterNote.all.each do |chatper_note|
-          chapter_file = "db/notes/chapters/#{chatper_note.section_id}_#{chatper_note.chapter_id.to_i}.yaml"
-          File.open(chapter_file, 'w') do |out|
-            chapter_doc = {
-              section: chatper_note.section_id,
-              chapter: chatper_note.chapter_id.to_i,
-              content: chatper_note.content.force_encoding('ASCII-8BIT').encode('UTF-8', undef: :replace, replace: ''),
-            }
-            YAML.dump(chapter_doc, out)
-          end
-        end
-      end
+  desc 'Import TARGET file'
+  task import: %i[environment class_eager_load] do
+    if ENV['TARGET'] && TariffSynchronizer::FileService.file_exists?(ENV['TARGET'])
+      Sequel::Model.subclasses.each(&:unrestrict_primary_key)
+      Sequel::Model.plugin :skip_create_refresh
+      dummy_update = OpenStruct.new(file_path: ENV['TARGET'], issue_date: nil)
 
-      desc 'Load Chapter notes into database'
-      task chapter_notes: :environment do
-        Dir[Rails.root.join('db/notes/chapters/*')].each do |file|
-          note = YAML.safe_load(File.read(file), [Symbol])
-          chapter_note = ChapterNote.find(section_id: note[:section],
-                                          chapter_id: note[:chapter].to_s) || ChapterNote.new(section_id: note[:section], chapter_id: note[:chapter].to_s)
-          chapter_note.content = note[:content]
-          chapter_note.save
-        end
-      end
-
-      desc 'Load measurment unit abbriviations'
```

Comment:
nice cleaning ... it looks like this task would have more sense as data migration